java/http/java-http-consumer/src/main/java/HttpConsumer.java

```diff
@@ -61,6 +66,8 @@ public static void main(String[] args) throws URISyntaxException, IOException, I
         }
 
         HttpConsumer consumer = new HttpConsumer(config, messagesReceivedLatch);
+        boolean enableAutoCommit = (props.getProperty(ENABLE_AUTO_COMMIT_CONFIG)) == null || Boolean.parseBoolean(props.getProperty(ENABLE_AUTO_COMMIT_CONFIG));
+        consumer.manualCommit = !enableAutoCommit;
```

Comment:
why do you need two variables with one the opposite of the other? You can just define the `enableAutoCommit` as the member field (instead of the `manualCommit`) but avoiding to do this negation.